library/aes.c

```diff
@@ -878,6 +878,37 @@ int mbedtls_internal_aes_encrypt(mbedtls_aes_context *ctx,
         uint32_t Y[4];
     } t;
 
+#if defined(MBEDTLS_AES_FEWER_TABLES)
+    /* This implementation is slower, but saves 516 bytes of code size */
+
+    if (MBEDTLS_IS_BIG_ENDIAN) {
```

Comment:
It's not always a compile-time constant, unfortunately

Reply:
Also, note 69604dcc532f6c6bf73b743b1aed426849d8dce3

Reply:
Ah, missed that, sorry

Reply:
Actually, I do have another PR to make it compile-time for IAR (currently IAR has to use the run-time fall-back because it doesn't define `__BYTE_ORDER__`). But that's out of scope for this PR.